core/templates/dev/head/pages/library-page/search-bar/search-bar.directive.html

```diff
@@ -120,7 +124,6 @@
     font-size: 15px;
     height: 34px;
   }
-
```

Comment:
Please add this blank line back. Ditto for the lines removed below.

Reply:
done